x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_indices.tsx

```diff
@@ -202,9 +201,6 @@ export const EngineIndices: React.FC = () => {
                 )}
               </EuiButton>
             </EuiFlexItem>
-            <EuiFlexItem>
```

Comment:
We should do this in another PR, we'll actually want to remove the entire `EuiFlexGroup` since it's not required without the Actions here anymore.